tests/integration/targets/ec2_ami/tasks/main.yml

```
@@ -1,35 +1,31 @@
 ---
-# tasks file for test_ec2_ami
+# Tests for ec2_ami
 
-- block:
+- name: 'Set module_defaults for ec2_ami tests'
+  module_defaults:
```

```suggestion
- module_defaults:
```
`name:` is for the block as a whole, and (while it doesn't actually do anything and is not attached to tasks or displayed anywhere) this name is inaccurate and could potentially confuse people.